docs/design/2021-04-21-unify-log-library.md

```diff
@@ -225,10 +225,10 @@ There must be something wrong with the engineering of these codes above, and the
 
 Rationale - for long-term consideration, we should maintain code quality. The speed of output can be sacrificed in time if necessary.
 
-To do these things:
+Implementation plan:
 
-1. Change the TiDB first, being dependent on the place to write dummy code.
-2. Modify the BR log to remove the dependency on TiDB `loguitl`.
-3. Modify PD and TiDB again to unify the log library to `pingcap/log`.
-4. BR, PD, TiDB all depend directly on `pingcap/log` and do not depend on any other log libraries (including `std/log`).
-5. BR, PD, and TiDB all encapsulate separate log layers on top of `pingcap/log` and do not rely on each other's log layers, preferring to add some boilerplate code.
+1. Unify log library for `pingcap/tidb` first. For dependent parts we write dummy code to satisfy.
```

Comment:
Will dummy code cause incorrect logics?

Reply:
I suspect codes related to logs will not have any tests, as they are very hard to be tested automatically in nature.

Reply:
@breeswish I think it is somehow a case link UI changes, that the developer verify locally and attach the output of log to show that after the change it looks good, while the reviewer can reproduce also.

By the way, we can call for an early testing effort for the log behavior among our developers to run their local tests in nightly master and thus we get potential regression converged.